src/collectors/hbase/hbase.py

```diff
@@ -1,110 +0,0 @@
-# coding=utf-8
-
-"""
-Diamond collector for HBase metrics, see:
```

Comment:
Why delete this? If we don't need it lets just not configure it as a collector?

Reply:
There is no reason anyone would ever want to use it; It's buggy and lacking functionality. Removing it improves the quality of the code by preventing anyone from mistakenly using it.

Reply:
@aaronabf as an opensource project, Diamond may have users of the older Hadoop Metrics Infra. I would prefer that we do not break compatibility.

We should ideally implement a new collector and leave the old one there - can be deprecated if the upstream maintainers agree on it.

Reply:
I would normally agree, but these collectors are straight-up buggy: they ignore documented configuration options and provide misleading timestamp info. Regardless, I wouldn't want to add this back. I would be fine having two collectors, for each metrics version, but these current collectors should be formally deprecated. We can have: `HadoopCollectorMetrics1` and `HadoopCollectorMetrics2`.